dbms/src/Storages/Page/V3/CheckpointFile/CPFilesWriter.h

```diff
@@ -46,6 +46,8 @@ class CPFilesWriter : private boost::noncopyable
          * lock files from `writeEditsAndApplyCheckpointInfo`.
          */
         const std::unordered_set<String> & must_locked_files = {};
+        UInt64 upload_sequence;
```

Comment:
```suggestion
        UInt64 sequence;
```
Better keep using the same name `sequence` inside `CheckpointFile`